packages/client-python/src/rocketride/mixins/execution.py

```
@@ -254,14 +255,20 @@ async def use(
 
         # Load pipeline configuration from file if needed
         if not pipeline:
-            with open(filepath, 'r', encoding='utf-8') as file:
-                # Prefer JSON5 for better developer experience (comments, trailing commas)
-                if json5:
-                    parsed = json5.load(file)
+
+            def _load_pipeline_config(path: str):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_load_pipeline_config`

(ANN202)